spec/integrations/applicant_spec.rb

```diff
@@ -23,6 +23,10 @@
         'town' => 'London',
         'postcode' => 'SW4 6EH',
         'country' => 'GBR'
+      },
```

Comment:
`phone_number` here as well

Reply:
In this one is the request body, so it's optional

Reply:
Updated!